.travis.yml

```diff
@@ -6,18 +6,20 @@
 #            - ubuntu-toolchain-r-test
 #        packages:
 #            - openvpn
-
-services:
-    - docker
+language: minimal
```

Comment:
Travis assumes Ruby by default and thus flagged builds for installing rvm and a default ruby. Using `minimal` reduces that overhead while still providing basic utilities like Docker out of the box.